pkg/callback/scheduler_callback.go

```diff
@@ -143,3 +144,35 @@ func (callback *AsyncRMCallback) ReSyncSchedulerCache(args *si.ReSyncSchedulerCa
 	}
 	return nil
 }
+
+// this callback implement scheduler plugin interface EventPlugin.
+func (callback *AsyncRMCallback) SendEvent(eventRecords []*si.EventRecord) error {
```

Comment:
Name is a bit confusing, is it SendEvent here?

Reply:
This function is part of the scheduler callback, it sends the `EventRecords` object to the shim. I think the `SendEvent` is concise and descriptive. What do you suggest?